qml/pages/ChatPage.qml

```diff
@@ -740,18 +740,16 @@ Page {
                                     Component {
                                         id: stickerPreviewComponent
                                         StickerPreview {
-                                            id: messageStickerPreview
                                             stickerData: ( display.content['@type'] === "messageSticker" ) ?  display.content.sticker : ""
-                                            visible: display.content['@type'] === "messageSticker"
-                                            anchors.horizontalCenter: parent.horizontalCenter
                                         }
                                     }
 
                                     Loader {
                                         id: stickerPreviewLoader
-                                        active: false
+                                        active: display.content['@type'] === "messageSticker"
```

Comment:
I suspect that this delay is responsible for annoying jumping and flicking during regular scrolling - delegates are initialized with wrong height and then get resized after a delay, it looks pretty chaotic, especially with images. I have a feeling that asynchronous loading and a bit of optimization could make both slow and fast scroll work reasonably well without this delay.

Reply:
Well, the jumping is also there if you load it immediately. To avoid that you probably need to reserve the respective space upfront, but I haven't digged deeper there yet. I also have the same issue with Piepmatz BTW.

Reply:
AFAICT the size of the image is known in advance, meaning that delegate can be sized right away, even before the actual content has been loaded.

Reply:
To a certain extent the height is known, yes. There is some uncertainty involved, but any precalculation is probably better than the large resizing procedures later. I introduced some changes in line with your suggestions on the Loader/Component handling with 67669cf63f15e04452cf51c3edfe9bc4f29c8ec5 and d3295bc49ac0276971507837e7410eebbedbbefb. Let's see how it performs...